### PR TITLE
Updated readme file

### PR DESCRIPTION
**Pull request by spurthihmbits**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,4 +84,36 @@ To play a random game:
   Following punctuation marks are to be supported – Comma Question mark, Full stop, Semicolon, Exclamation.
   Do this for a file and maintain sequences of lines.
   On executing the program, it should prompt the user to enter input file name and generate an output file with scrambled text. The output file should be named by   appending the word “Scrambled” to the input file name.
+<<<<<<< HEAD
```
Comment (kartik70rajesh) [inline README.md]: take out this line

File: README.md [hunk 2]
```diff
@@ -84,4 +84,36 @@ To play a random game:
...
+Below are the three features that our screen pet will perform:
+
+Happy Face: When you will click on it with the mouse the screen pet will blush and it will look happy.
+
+Cheeky Face: If you double-click it, then it will show its tongue to you.
+
+Sad Face: If you ignore it then it will become sad and the point where you click then it will become happy again.
+  
+=======
+>>>>>>> upstream/main
```
Comment (kartik70rajesh) [inline README.md]: take out this line

File: README.md (hunk 1 above)
Comment (spurthihmbits) [reply README.md]: updated

File: README.md (hunk 2 above)
Comment (spurthihmbits) [reply README.md]: updated

**Pull request merged**